getblurryscale() undefined error resolved AND changing LBL syntax

sagarpreet-chadha opened the pull request:
Fixes #194 , #193 😄 .

Work in Progress 🚧 👷 !

sagarpreet-chadha commented:
Hi @jywarren ,
This PR changes the syntax of LBL declaration to the same way a standard Leaflet object is added .

`var map123 = L.map('map').setView([23,77] , 3) ;`

`var blurredLocation = new BlurredLocation(options) ;`
` blurredLocation.addTo(map123)  ;`

Kindly review 😄 !

jywarren commented:
Awesome! Let's update the docs too!

jywarren commented:
Is this a breaking change regarding https://semver.org?

jywarren commented:
If not, let's merge! If so, maybe we need to update the version number first. Thanks!!!

igniteeng000 commented:
@jywarren I am uploading this gif to my proposal but was unable to do so. any idea??
![crdtpng](https://user-images.githubusercontent.com/25272573/55765107-ca26b880-5a8b-11e9-8429-a8dc7474c075.gif)

jywarren commented:
If you upload to Github you should be able to copy the URL to your proposal!

On Mon, Apr 8, 2019 at 8:24 PM Ashutosh Kumar <notifications@github.com>
wrote:

> @jywarren <https://github.com/jywarren> I am uploading this gif to my
> proposal but was unable to do so. any idea??
> [image: crdtpng]
> <https://user-images.githubusercontent.com/25272573/55765107-ca26b880-5a8b-11e9-8429-a8dc7474c075.gif>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/publiclab/leaflet-blurred-location/pull/195#issuecomment-481055078>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABfJ6fqB6UdenuXXW3hRSuNdPjf5I5lks5ve92sgaJpZM4chaTI>
> .
>

igniteeng000 commented:
@jywarren is this gif fine for demo implementation of synchronous editing.

sagarpreet-chadha commented:
Hi @jywarren ,  updated readme and version number is already in the PR 😄 . 

I have changed the version number from `1.2.8` to `1.3.0` but i guess we are introducing backwards incompatible change , so we should change version to `2.0.0` according to https://semver.org/ . 

Thank you !

jywarren commented:
+1 `v2.0`!!! Thank you!!!

sagarpreet-chadha commented:
Done 👍 . Should we merge this ? Thanks You !
